app/styles/front.scss

```diff
@@ -306,17 +306,15 @@ $autre-lighter: #cce4e7;
   border-left-width: 4px;
   margin-bottom: 10px;
 
-  h2 {
```

Comment:
Does this h2 block apply to all h2 throughout the app ? It's inside `@mixin ressource-category-capture` block, so I thought it only applied on a restricted area.